Tests/SwiftCliCoreTests/ChartifyTest.swift

```diff
@@ -58,3 +70,15 @@ class MockFileValidator: FileValidator {
         return ["k1"]
     }
 }
+
+class MockWriteToFile: FileWriter {
+    var wasWriteToFileCalled = false
+
+    override func writeFile(chart: String,
+                            filePath: String,
+                            fileName: String) {
+
+        wasWriteToFileCalled = true
```

Comment:
Having naming consistency, e.g. `wasWriteFileCalled`, makes it clearer which method you're checking.